python/README.rst

```diff
@@ -132,6 +137,35 @@ example file:
 For any ``.ini`` setting you can use an environment variable instead. It takes the form of
 ``LOOKERSDK_<UPPERCASE-SETTING-FROM-INI>`` e.g. ``LOOKERSDK_CLIENT_SECRET``
 
+A final option is to provide your own implementation of the ApiSettings class. In particular you may want to override the ``read_config`` function. Example:
+
+
+.. code-block:: python
+
+    import os
+    import looker_sdk
+    from looker_sdk import api_settings
+
+    class MyApiSettings(api_settings.ApiSettings):
+        def __init__(self, *args, **kw_args):
+            self.my_var = kw_args.pop("my_var")
+            super().__init__(*args, **kw_args)
+
+        def read_config(self) -> t.Dict[str, str]:
+            config = super().read_config()
```

Comment:
```suggestion
            # Return dict must include at least `client_id` and `client_secret` keys
            config = super().read_config()
```

Reply:
can you also add this comment to the `api_settings.ApiSettings.read_config` method if it's not clearly indicated there? This would be in addition to updating the `api_settings.PApiSettings.read_config` declaration I already mentioned